src/OrchardCore.Modules/OrchardCore.Html/Handlers/HtmlBodyPartHandler.cs

```diff
@@ -45,7 +46,7 @@ public override Task GetContentItemAspectAsync(ContentItemAspectContext context,
                     templateContext.MemberAccessStrategy.Register<HtmlBodyPartViewModel>();
                     templateContext.SetValue("Model", model);
 
-                    var result = await _liquidTemplateManager.RenderAsync(part.Html, HtmlEncoder.Default, templateContext);
+                    var result = await _liquidTemplateManager.RenderAsync(part.Html, _htmlEncoder, templateContext);
                     bodyAspect.Body = _bodyAspect = new HtmlString(result);
                 }
```

Comment:
How will this work (noting that it's no different to what we do already) with the Lucene full text aspect?

which uses this `bodyAspect.Body.WriteTo(sw, NullHtmlEncoder.Default);` but chances are good the `BodyAspect` will be coming through encoded by rendering any liquid in it?

Reply:
`bodyAspect.Body.WriteTo(sw, NullHtmlEncoder.Default);` means that it won't encode the html, so we'll keep the html as is.